src/scripts/Tara.deploy.s.sol

```diff
@@ -57,7 +59,8 @@ contract TaraDeployer is Script {
 
         TaraBridge taraBridge = new TaraBridge{value: 2 ether}(
             taraAddress,
-            IBridgeLightClient(address(client))
+            IBridgeLightClient(address(client)),
+            FINALIZATION_INTERVAL
```

Comment:
Lets have it before the contract creation as in Eth.deploy, so you won't be jumping to top of the file to modify it